dev/pages/Elements.vue

```diff
@@ -1,8 +1,8 @@
 <script setup lang="ts">
 import { ref } from "vue"
-import ColorRow from "../../dev/helpers/ColorRow.vue"
+import ColorRow from "../components/ColorRow.vue"
```

Comment:
It would be nice to remove `max-w-3xl` from all of our page containers to get more horizontal space.